.github/workflows/release.yml

```diff
@@ -30,6 +30,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Count lines of code (CLOC)
```

Comment:
What is the point of having this?

Reply:
Wanted for a presentation that Apache is giving.